### PR TITLE
15802 change table anchor color

### PR DESCRIPTION
**Pull request by arthanson**
### Fixes: #15802 

Made a lighter version of the current link color for tables

Monosnap Devices | NetBox 2024-04-29 10-22-58

### CODE REVIEW

**Review by jeremystretch — CHANGES_REQUESTED** [review]
> However it doesn't have any contrast between a link and a non-link (see screenshot) not sure if we want this or to go with a different tabler link color...

We definitely still want to render links in a different color. #15802 is just to increase the contrast of links under dark mode.

**Review by jeremystretch — CHANGES_REQUESTED** [review]
The color change should only apply to dark mode.

screenshot

**Comment by arthanson** [comment]
I tried using different media queries but they didn't work properly when switching between light and dark, even specifically defining media queries for both.  As a compromise I picked a filter to slightly lighten the link which seems to look fairly good in both light and dark.

File: netbox/project-static/styles/overrides/_tabler.scss [hunk 1]
```diff
@@ -27,3 +27,7 @@ pre {
   // Tabler sets display: flex
   display: inline-block;;
 }
+
+table a {
+  filter: brightness(110%);
```
Comment (jeremystretch) [inline netbox/project-static/styles/overrides/_tabler.scss:32]: Please add a comment explaining why the rule exists.

**Review by jeremystretch — CHANGES_REQUESTED** [review]

**Review by jeremystretch — APPROVED** [review]

**Pull request merged**